tasks/variables-spatialite.yml

```diff
@@ -0,0 +1,26 @@
+- name: Set spatialite_path for Fedora
+  set_fact:
+    openwisp2_spatialite_path: "mod_spatialite"
+  when: ansible_distribution == 'Fedora'
```

Comment:
Fedora doesn't have mod-spatialite, I think you can remove this (`null` will be used on RedHat based systems).

Reply:
I ran it on my system again, it looks like in fedora-27, it requires `mod_spatialite`, it fails otherwise!
Should i try on some other fedora version as well?

Reply:
I just checked with the version on anisble-galaxy, that's failing because of `mod_spatialite` too!
(On fedora 27)

Reply:
As per my understanding right now, the value of `openwisp2_spatialite_path` is required according to the table below:

OS | Version |
--- | --- | 
Ubuntu:14.04 | false
Ubuntu:16.04 | mod_spatialite
Ubuntu:18.04 | mod_spatialite.so
Debian:8 | false
Debian:9 | mod_spatialite
Debian:10 | mod_spatialite.so
Fedora:27 | mod_spatialite

Reply:
ok if you tested it on Fedora27 and it's required let's leave it